examples/solid-client-access-grants-demo/src/utils/getEnvironment.ts

```diff
@@ -40,8 +40,8 @@ export function getEnvironment(): Environment {
       process.env.ACCESS_MANAGEMENT_APP ??
         "https://podbrowser.inrupt.com/privacy/access/requests/"
     ),
-    clientId: process.env.CLIENT_ID ?? "",
-    clientSecret: process.env.CLIENT_SECRET ?? "",
+    clientId: process.env.E2E_TEST_REQUESTOR_CLIENT_ID ?? "",
+    clientSecret: process.env.E2E_TEST_REQUESTOR_CLIENT_SECRET ?? "",
```

Comment:
I don't think this is correct? I think the example has (or should have) it's own `.env` file